scapy/contrib/gtp.py

```diff
@@ -342,8 +343,10 @@ class IE_EndUserAddress(IE_Base):
                     BitField("SPARE", 15, 4),
                     BitField("PDPTypeOrganization", 1, 4),
                     XByteField("PDPTypeNumber", None),
-                    ConditionalField(IPField("PDPAddress", RandIP()),
-                                     lambda pkt: pkt.length > 2)]
+                    ConditionalField(IPField("PDPAddress_IPv4", RandIP()),
+                                     lambda pkt: pkt.length == 6 or pkt.length == 22),
+                    ConditionalField(IP6Field("PDPAddress_IPv6", "::1"), lambda
+                                    pkt: pkt.length == 18 or pkt.length == 22)]

```

Comment:
Agreed. Also, why choose `"::1"` as default value when the IPv4 counterpart has `RandIP()`?